### DISCUSSION TITLE
NFT

### DISCUSSION THREAD

**Discussion by cstr87** [community/community#38870] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi guys!  I make some NFTs in Visual Studio Code and I have a big question:
I rename my files in the folder with #number for rarity ( example: colorredbckg#100 or colorbluebckg#50 )
I make a collection of 159 images, because I have unique 159 characters for the images, everything works fine, the rarity of colors, background colors...(I have between 15-20 colors background), but...my unique 159 characters doesn't fill all of them in the generated images. what can I do to appear all of them? I can rename the file to "examplename#1000" or "#10000" for a permanent guarantee chance to fill or let the normal name of the file without a #number, or something else"? Thank you!!

**Comment by modster** [comment]
I suggest you hire a consultant to sit with you at the computer and show you how to fix your bug. After that you won't have this problem ever again.

Merry Christmas
🎅🎄🤶
